src/test/java/com/adobe/target/edge/client/ondevice/DefaultRuleLoaderTest.java

```diff
@@ -184,7 +187,7 @@ void testDefaultRuleLoader() {
         .when(defaultRuleLoader)
         .executeRequest(any());
 
-    defaultRuleLoader.start(clientConfig);
+    defaultRuleLoader.start(clientConfig, telemetryService);
```

Comment:
As part of this test case would it be difficult to add an assertion that the artifact download telemetry entry gets added to the entry store?